docs/framework/windows-workflow-foundation/samples/basic-validation.md

```diff
@@ -23,7 +23,7 @@ This sample consists of an activity, `CreateProduct`, which validates that its `
   
 -   Create a constraint (`PriceGreaterThanCost`). This is where all the validation logic resides.  
   
--   Override <xref:System.Activities.CodeActivity%601.OnGetConstraints%2A> and add the constraint (`PriceGreaterThanCost`) to the constraints <xref:System.Collections.IList>.  
+-   Override `System.Activities.CodeActivity.OnGetConstraints()` and add the constraint (`PriceGreaterThanCost`) to the constraints <xref:System.Collections.IList>.  
```

Comment:
If we have it as a note, it's ok as well... as long as we're tracking this somewhere.